### PR TITLE
Ui improvement

### PR DESCRIPTION
**Pull request by Hongzred**
- make it able to toggle side navs in chatbox component from menu button in header component
- make chatroom list and user list overflow scrollable

### CODE REVIEW

**Comment by bidachen** [comment]
image
Is it possible to hide the scroll bar? I think it makes the web looks bad. lol

**Comment by Hongzred** [comment]
I don’t have this kind of scroll bar in my browser lol. I think windows
doesn’t support the scroll bar in material design

File: src/app/header/header.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <mat-toolbar color="black">
     <button mat-icon-button>
-      <mat-icon>menu</mat-icon>
+      <mat-icon (click)="toggleSideNav()">menu</mat-icon>
```
Comment (angelica-hernandez) [inline src/app/header/header.component.html]: Maybe change the icon to something more related to the function of it?

Comment (Hongzred) [reply src/app/header/header.component.html]: Oh, I didnt know we are going to remove the right sidenav. If so, where do we view the friend list?
I looked over the material icons, haven't find something that would replace the current one lol
And any suggestion about the icon?

**Pull request closed without merging**